src/components/note-modal/label-modal/LabelModal.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
No need to Import React in the latest Versions.

Reply:
okay